### DISCUSSION TITLE
nofee-ng is working in standalone mode bu no works in SparkPool

### DISCUSSION THREAD

**Discussion by minerpxe** [nofee-ng/nofee-ng#21] [Q&A]
I install  some works .
this  is ok, i chang walk is my walk .
systemctl status nofee  is ok

I test is usd  Gminer 2.58 

but SparkPool  is no my works 

the works  shi show   five DevFee found

**Comment by nofee-ng** [comment]
How long did you run nofee? It may get some time for the refunded devfee show up

**Reply by minerpxe** [reply]
It's work one week, i have  115 pc . bu not no date in SparkPool 


Last login: Fri Mar 12 20:20:09 2021 from 192.168.3.144
kg@minerpxe:~$ systemctl status nofee
● nofee.service - Next Generation NoFee Miner
   Loaded: loaded (/opt/nofee-ng/nofee.service; enabled; vendor preset: enabled)
   Active: active (running) since Sun 2021-07-04 09:41:57 UTC; 2h 59min ago
 Main PID: 4378 (nofee_eth)
    Tasks: 1 (limit: 4915)
   CGroup: /system.slice/nofee.service
           └─4378 /opt/nofee-ng/nofee_eth -rs

Jul 04 09:41:57 minerpxe systemd[1]: Started Next Generation NoFee Miner.
Jul 04 09:41:57 minerpxe nofee_eth[4378]: Found NoFee Wallet: 0x6de55b80c537d8a4
Jul 04 09:42:01 minerpxe nofee_eth[4378]: Running in mode: 3.
Jul 04 09:42:01 minerpxe nofee_eth[4378]: !!!!! DO NOT kill this program with 'k
Jul 04 09:42:17 minerpxe nofee_eth[4378]: DevFee found
Jul 04 10:12:23 minerpxe nofee_eth[4378]: DevFee found
Jul 04 10:42:35 minerpxe nofee_eth[4378]: DevFee found
Jul 04 11:12:41 minerpxe nofee_eth[4378]: DevFee found
Jul 04 11:42:50 minerpxe nofee_eth[4378]: DevFee found
Jul 04 12:12:59 minerpxe nofee_eth[4378]: DevFee found

**Comment by minerpxe** [comment]
kg@minerpxe:~$ systemctl status nofee
● nofee.service - Next Generation NoFee Miner
   Loaded: loaded (/opt/nofee-ng/nofee.service; enabled; vendor preset: enabled)
   Active: active (running) since Sun 2021-07-04 09:41:57 UTC; 3h 0min ago
 Main PID: 4378 (nofee_eth)
    Tasks: 1 (limit: 4915)
   CGroup: /system.slice/nofee.service
           └─4378 /opt/nofee-ng/nofee_eth -rs

Jul 04 09:41:57 minerpxe systemd[1]: Started Next Generation NoFee Miner.
Jul 04 09:41:57 minerpxe nofee_eth[4378]: Found NoFee Wallet: 0x6de55b80c537d8a4597a28038514b74b91eb3037
Jul 04 09:42:01 minerpxe nofee_eth[4378]: Running in mode: 3.
Jul 04 09:42:01 minerpxe nofee_eth[4378]: !!!!! DO NOT kill this program with 'kill -9' !!!!!
Jul 04 09:42:17 minerpxe nofee_eth[4378]: DevFee found
Jul 04 10:12:23 minerpxe nofee_eth[4378]: DevFee found
Jul 04 10:42:35 minerpxe nofee_eth[4378]: DevFee found
Jul 04 11:12:41 minerpxe nofee_eth[4378]: DevFee found
Jul 04 11:42:50 minerpxe nofee_eth[4378]: DevFee found
Jul 04 12:12:59 minerpxe nofee_eth[4378]: DevFee found

**Reply by nofee-ng** [reply]
What about your main pool? Did you check it?

**Reply by minerpxe** [reply]
pool is  stratum+tcp://ethash.poolbinance.com:1800 and  eth-pool.beepool.org:9530

**Reply by minerpxe** [reply]
now i test the pool is eth.2miners.com.it is work very nice
I have a  question , nofee is work in miniZ,or nbminer ?
